extensions/amp-pinterest/0.1/follow-button.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Util} from './util';
+
+// Popup options
+const POP_FOLLOW = `status=no,resizable=yes,scrollbars=yes,
+  personalbar=no,directories=no,location=no,toolbar=no,
```

Comment:
The popup can invoke the login window if they attempt to follow and are not logged in. Our accepted practice is as written here.